### ISSUE TITLE
webpack-dev-server 3.8  broken on windows 10

### ISSUE THREAD

**Issue by docteurzoidberg** [vuejs/vue-cli#4424]
### Version
3.10.0

### Reproduction link
[npm run serve](npm run serve)

### Environment info
```
Environment Info:

  System:
    OS: Windows 10
    CPU: (4) x64 Intel(R) Core(TM) i5-7300U CPU @ 2.60GHz
  Binaries:
    Node: 10.16.1 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.17.3 - C:\Users\DrZoid\AppData\Roaming\npm\yarn.CMD
    npm: 6.10.3 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Edge: 44.18362.1.0
  npmPackages:
    @vue/babel-helper-vue-jsx-merge-props:  1.0.0
    @vue/babel-plugin-transform-vue-jsx:  1.0.0
    @vue/babel-preset-app:  3.10.0
    @vue/babel-preset-jsx:  1.1.0
    @vue/babel-sugar-functional-vue:  1.0.0
    @vue/babel-sugar-inject-h:  1.0.0
    @vue/babel-sugar-v-model:  1.0.0
    @vue/babel-sugar-v-on:  1.1.0
    @vue/cli-overlay:  3.10.0
    @vue/cli-plugin-babel: ^3.10.0 => 3.10.0
    @vue/cli-plugin-eslint: ^3.10.0 => 3.10.0
    @vue/cli-service: ^3.10.0 => 3.10.0
    @vue/cli-shared-utils:  3.10.0
    @vue/component-compiler-utils:  2.6.0 (3.0.0)
    @vue/preload-webpack-plugin:  1.1.1
    @vue/web-component-wrapper:  1.2.0
    eslint-plugin-vue: ^5.0.0 => 5.2.3 (4.7.1)
    vue: ^2.6.10 => 2.6.10
    vue-eslint-parser:  5.0.0 (2.0.3)
    vue-hot-reload-api:  2.3.3
    vue-loader:  15.7.1
    vue-style-loader:  4.1.2
    vue-template-compiler: ^2.6.10 => 2.6.10
    vue-template-es2015-compiler:  1.9.1
  npmGlobalPackages:
    @vue/cli: Not Found
```

### Steps to reproduce
"npm run serve" on newly create project with "vue create" (default options)

### What is expected?
'npm run serve' should work

### What is actually happening?
'npm run serve' give this error: 

 INFO  Starting development server...
 10% building 2/2 modules 0 active ERROR  Error: Watching remote files is not supported.
Error: Watching remote files is not supported.
at Server.setupWatchStaticFeature

---
"it worked last time !"

**Comment by sodatea** [comment]
Can't reproduce. Please provide a reproduction repo.

And, what file system are you on? What's your project path (run `pwd` in the terminal and paste the output here)?

**Comment by docteurzoidberg** [comment]
pwd = c:\dev\some-repo

Doesn't need a repo, i just created the app from vue-cli with nothing in it (vue cli did the npm install)

**Comment by docteurzoidberg** [comment]
oh no, don't tell me it's not because of my "dev" folder :D

**Comment by docteurzoidberg** [comment]
It's the pwd thing ! i just did the project in another folder and it works flawlessly :o

**Comment by sodatea** [comment]
False-positive in https://github.com/sindresorhus/is-absolute-url (webpack-dev-server changed from testing `/https?:/` to using this library in the 3.8 release)

Please report to that repository.

I believe it's because of the lower case `c`. But how come the lower case volume label? I thought it would always be UPPER CASE.

**Comment by elchinzadeh** [comment]
I removed space from the parent folder name and it worked without problem

**Comment by NateScarlet** [comment]
> I thought it would always be UPPER CASE

Windows uses case insensitive path. `c:\` is equivalent to `C:\`.

**Comment by docteurzoidberg** [comment]
> 
> 
> False-positive in https://github.com/sindresorhus/is-absolute-url (webpack-dev-server changed from testing `/https?:/` to using this library in the 3.8 release)
> 
> Please report to that repository.
> 
> I believe it's because of the lower case `c`. But how come the lower case volume label? I thought it would always be UPPER CASE.

maybe vue-cli should downgrade the version of webpack-dev-server 3.8  to 3.7.2 until it's fixed in is-absolute-url ?

**Comment by viruscamp** [comment]
I used ProcessHacker to inspect the process property of node.exe, and found when the Current Directory was 'e:/abc/proj', the issue happened, when is was 'E:/abc/proj', it was ok.
I ran 'npm run serve' from a standalone cmd, it was ok. Current  Directory was 'E:/abc/proj'
I ran 'npm run serve' from an embedded cmd of VS Code, it was ok. Current  Directory was 'E:/abc/proj'
I met the issue only  I started debugging from VS Code debug launch. Current  Directory was 'e:/abc/proj'
